doc/classes/TextServer.xml

```diff
@@ -1155,6 +1155,9 @@
 		<constant name="GRAPHEME_IS_SPACE" value="8" enum="GraphemeFlag">
 			Grapheme is whitespace.
 		</constant>
+		<constant name="GRAPHEME_IS_PUNCTUATION" value="16" enum="GraphemeFlag">
```

Comment:
The value should be 256 no?

Reply:
Unless you reorder them, in which case the later enums should change value too.

Reply:
Should be 256, fixed.